### PR TITLE
Cy 247 add hidden value secret

### PR DESCRIPTION
**Pull request by inbala**

### CODE REVIEW

File: content/cli/orch_cli/secrets.md [hunk 1]
```diff
@@ -36,17 +36,19 @@ One of these flags:
 
 #### Optional flags:
 
-* `-u, --update-if-exists` - Update secret with new value if it already exists
+* `-u, --update-if-exists` - Update secret value if secret key already exists. [This option is deprecated; use cfy secrets update command instead]. You cannot use this argument with arguments: [visibility, hidden_value]
 * `-l, --visibility TEXT` - Defines who can see the resource, can be set to one of ['private', 'tenant', 'global'] [default: tenant].
+* `--hidden-value` - The secret value is only shown to the user that created the secret and to admins. Use of the secret is allowed according to user roles and the visibility of the secret.
```
Comment (talisela) [inline content/cli/orch_cli/secrets.md]: Let's add the tenant admins as well ("The secret value is only presented to the user who created it, tenant managers and sys-admins") so there's no misunderstandings

File: content/cli/orch_cli/secrets.md [hunk 2]
```diff
@@ -160,17 +164,23 @@ Update an existing secret.
 One of these flags:
 
 * `-s, --secret-string TEXT` - The string to use as the secret's value.
-* `-f, --secret-file TEXT` - The name of the secret file that contains the value to be set.
+* `-f, --secret-file TEXT` - The secret's file to use its content as value to be set.
+                             You cannot use this argument with arguments: [secret_string].
+
+#### Optional flags:
 
+* `--hidden-value / --not-hidden-value` - The secret value is only shown to the user that created the secret and to admins. Use of the secret is allowed according to user roles and the visibility of the secret.
```
Comment (talisela) [inline content/cli/orch_cli/secrets.md]: The secret's value is only presented to the user who created it, tenants managers and sys-admins. Use of the secret is allowed according to the users roles, and visibility of the secret.

File: content/working_with/manager/using-secrets.md [hunk 3]
```diff
@@ -6,10 +6,18 @@ draft: false
...
+The secrets store provides a secured variable storage (key-value pairs) for data that you do not want to expose in plain text in Cloudify blueprints, such as login credentials for a platform.
+The secrets values are being stored encrypted in our DB. We use the Fernet encryption algorithm of cryptography library,
+it's a symmetric encryption method which makes sure that the message encrypted cannot be manipulated/read without the key.
+When you create a secret, the key value can be a text string or it can be a file that contains the key value. With the secrets store, you can make sure all secrets (for example credentials to IaaS environments) are stored separately from blueprints, and adhere to isolation requirements between different tenants. You can include the secret's key in your blueprints and not include the actual values in the blueprints.
 For more information, see the [get_secret]({{< relref "developer/blueprints/spec-intrinsic-functions.md#get-secret" >}}) intrinsic function.
 
+### Hidden value secret
+
+All the secrets values are encrypted in the DB, when you create a secret you can specify if you want it’s value to be hidden.
```
Comment (talisela) [inline content/working_with/manager/using-secrets.md]: All the secrets values are encrypted in the DB. When you create a secret you can specify if you want its value to be hidden or not.

File: content/working_with/manager/using-secrets.md [hunk 4]
```diff
@@ -6,10 +6,18 @@ draft: false
...
+A hidden-value secret means the secret value is only shown to the user that created the secret and to admins.
+Use of the secret is allowed according to user roles and visibility setting for the secret.
+
```
Comment (talisela) [inline content/working_with/manager/using-secrets.md]: A hidden-value secret means the secret's value will only be presented to the user who created it, tenant managers and sys-admins. Use of the secret is allowed according to the users roles, and visibility of the secret.

**Comment by bmansheim** [comment]
@inbala You have a comment from Tali on this. Also, the PR has a lot of unrelated stuff in it. I recommend that you cherry-pick the commit to a new branch and do the PR from there.

**Pull request closed without merging**